src/App.jsx

```diff
@@ -14,70 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelected]
+    = useState(goods.find(item => item === 'Jam'));
```

Comment:
1. you can just use 'Jam' as a default value
2. name setter according to getter
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```